src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
will be better to set these styles using separate class instead of using tag selector